docs/Información del servicio/Story board.mdx

```diff
@@ -0,0 +1,124 @@
+## Historial de versiones
+
+| Versión | Fecha      | Diferencia entre versiones                                                                                     |
+| ------- | ---------- | ---------------------------------------------------------------------------------------------------------------|
+| 1.0     | 2024-03-14 | Versión inicial del documento.                                                                                 |
+                                               |
+
+## StoryBoard CyC
+
+
+### 1.1 Inicio
+![Inicio](/img/storyBoard/Cell-Row-0-Col-0.png)
+
+Inicio referenciando a los simpsons como introducción.
+
+### 1.2 Frente a la sede de CyC
+![Frente a la sede de Cirio y Costal](/img/storyBoard/Cell-Row-0-Col-1.png)
+
+En frente de la sede de cirio y costal se encuentran varios personajes de los simpsons esperando la cola para poder recibir 
+alimentos y otros recursos.
+
+
+### 1.3 Homer pide su pedido
+![Homer pide su pedio](/img/storyBoard/Cell-Row-0-Col-2.png)
+
+Homer Simpson pide los alimentos que solicitó hace un par de días y nuestro Avatar AL se dispone a buscarlos.
+
+
+### 1.4 Error en Excel
+![Error en excel](/img/storyBoard/Cell-Row-0-Col-3.png)
+
+Al abre el excel y se encuentra en una situación comprometida ya que el excel dejo de funcionar.
```

Comment:
The case you present is good although it is not very common, I recommend you to use other excel disadvantages such as a sheet with a lot of columns and rows.